src/lib/metadata/fetch.ts

```diff
@@ -102,14 +101,12 @@ const buildTokenMetadataFromFetched = (
         artifactUri: token.artifactUri
       };
 
-export const loadWhitelist$ = (selectedRpc: string): Observable<TokenMetadata[]> =>
-  isDcpNode(selectedRpc)
-    ? of([])
-    : fetchWhitelistTokens$().pipe(
-        map(tokens =>
-          tokens.map(token => transformWhitelistToTokenMetadata(token, token.contractAddress, token.fa2TokenId ?? 0))
-        )
-      );
+export const loadWhitelist$ = (): Observable<TokenMetadata[]> =>
+  fetchWhitelistTokens$().pipe(
```

Comment:
i think you will see tezos whitelisted tokens on a new wallets when changing network to dcp without isDcpNode check.. Did you test that?

Reply:
ok